2017-bae-europe/eu_future/eu_future.ipynb

```diff
@@ -95,7 +93,7 @@
    "source": [
     "### The output file should be named \n",
     "         \n",
-    "          cyclus.sqlite"
+    "          eu_future.sqlite"
```

Comment:
If it were a variable at the top of the notebook, it would be pointing to an sqlite file that hasn't been generated yet, which might cause confusion.